### ISSUE TITLE
Pragma once is ignored

### ISSUE THREAD

**Issue by Nielsbishere** [tgjones/HlslTools#247]
#pragma once should allow duplicate includes to be ignored. Currently it'll think the same symbols are defined multiple times if one of the includes includes the same include as a different include. The error is about ambiguous symbols.

**Comment by BenSimsTantalus** [comment]
This would be great, especially because Unreal's shaders use #pragma once everywhere, and HLSL tools is now included in the latest Visual Studio previews along with improved Unreal support. I think this also causes a lot of slowdowns as it means the same files are processed many times.

**Comment by tgjones** [comment]
I agree, this would be a nice feature to add. PRs gratefully accepted 😄
